src/pages/HomePage.tsx

```
@@ -288,98 +387,185 @@ export const HomePage: React.FC = () => {
     <>
       <section id="introduction">
         <div className="container">
-          <h2 className="welcome-text">Hey, I'm Tom<span className="cursor-blink">▮</span></h2>
-          <br />
-          <h2>// A Full Stack AI Engineer from the UK</h2>
-          <h2>With a passion for cutting-edge technology and creative problem-solving, I'm here to help transform your digital visions into reality.</h2>
-          <h2 className="introduction-h2">My expertise spans web development, AI integration, and innovative design solutions.</h2>
+          <h2 className="welcome-text">Hey, I'm Tom</h2>
+          <h2 className="hero-tagline">
+            <span className="hero-tagline-prefix">// I build</span>
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)